metadata_layer.py

```diff
@@ -1093,7 +1102,7 @@ def mirror_link(self, pool, data_source, link, link_obj, policy):
             if pool and pool.edition and pool.edition.title:
                 title = pool.edition.title
             else:
-                title = None
+                title = self.title or None
```

Comment:
I like this in theory but it feels like this is going to rename all our Project Gutenberg EPUBs that we've already stored on S3, and it's going to be a pain to rename or re-upload all of them. Do you have any ideas for this?